documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p>Our docs need to be structured with a single H1 at the top - the title. Other headings start from H2.</p><p> </p><p>Also I think it'd be clearer to have this section be "Use your model to make predictions". Using the test set makes sense, but pedagogically the user really just needs to know: 1) load data, 2) train model, 3) understand model fitness and now 4) use the model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>